.gitignore

```diff
@@ -1,5 +1,10 @@
-
 venv/
 .idea/
-
 __pycache__/
+RunOnCluster.egg-info/
+*/__pycache__/
+setup.py
+loadimagesfromomero.py
+CellProfiler-plugins/
+__init__.py
+*.pyc
```

Comment:
Missing newline at end of file. (I won't comment on every file where this is a problem, but please check all files for this problem and fix where it occurs.)